<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SimpleIMContactManager</class>
 <widget class="QWidget" name="SimpleIMContactManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QLabel" name="icqLabel" >
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/icq_off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>ICQ ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2" >
    <widget class="QLineEdit" name="icqLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Update your profile</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>354</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" colspan="2" >
    <widget class="QLineEdit" name="yahooLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Update your profile</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QLabel" name="label_9" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Jabber/GoogleTalk ID:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QLabel" name="sipLabel" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/sip.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>SIP ID:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" >
    <widget class="QLineEdit" name="jabberLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Update your profile</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Yahoo! ID:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QLabel" name="jabberLabel" >
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/jabber_off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="wengoLabel" >
     <property name="maximumSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/wengo.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="msnLabel" >
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/msn_off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="aimLabel" >
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/aim_off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="yahooLabel" >
     <property name="pixmap" >
      <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/protocols/yahoo_off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>@company@ ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>MSN ID:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>AIM ID:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2" >
    <widget class="QLineEdit" name="aimLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Update your profile</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2" >
    <widget class="QLineEdit" name="msnLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Update your profile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2" >
    <widget class="QLineEdit" name="wengoLineEdit" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>exemple: username@server</string>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="indent" >
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2" >
    <widget class="QLineEdit" name="sipLineEdit" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>wengoLineEdit</tabstop>
  <tabstop>msnLineEdit</tabstop>
  <tabstop>aimLineEdit</tabstop>
  <tabstop>yahooLineEdit</tabstop>
  <tabstop>jabberLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
